Dockerfile

```diff
@@ -24,12 +26,15 @@ RUN dnf -y install \
 FROM base as builder
 WORKDIR /src
 COPY . .
+
+# Keep the list of dependencies alphabetically sorted
 RUN dnf -y install \
     --setopt install_weak_deps=0 \
     --nodocs \
-    gcc \
-    # not a build dependency, but we copy the binary to the final image
+    # cargo is not a build dependency, but it pulls in the Rust compiler & library adding >200M
+    # of installed size which is excessive when we only need a single binary which we'll copy over
```

Comment:
We don't need it _yet_. Once we get to building Python dependencies with `rustc` we'll need these. Obligatory not a blocker, will be dealt separately.

Reply:
Oh no! ... Also, not entirely unexpected :)
I'll drop this hunk then since the effort you mention will land in the foreseeable future.